examples/appliance_snmp_v3_user.go

```diff
@@ -0,0 +1,111 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/HewlettPackard/oneview-golang/ov"
+)
+
+func main() {
+	var (
+		ClientOV *ov.OVClient
+	)
+	apiversion, _ := strconv.Atoi(os.Getenv("ONEVIEW_APIVERSION"))
+	ovc := ClientOV.NewOVClient(
+		os.Getenv("ONEVIEW_OV_USER"),
+		os.Getenv("ONEVIEW_OV_PASSWORD"),
+		os.Getenv("ONEVIEW_OV_DOMAIN"),
+		os.Getenv("ONEVIEW_OV_ENDPOINT"),
+		false,
+		apiversion,
+		"*")
+
+	snmpv3User := ov.SNMPv3User{
+		UserName:                 "user47",
+		SecurityLevel:            "Authentication and privacy",
+		AuthenticationProtocol:   "SHA1",
+		AuthenticationPassphrase: "authPass",
+		PrivacyProtocol:          "AES-128",
+		PrivacyPassphrase:        "1234567812345678",
+	}
+
+	//Creating an SNMPv3 User
+	_, err := ovc.CreateSNMPv3Users(snmpv3User)
+	if err != nil {
+		panic(err)
+	} else {
+		fmt.Println("#-----Created SNMPv3 User-----#")
+	}
+
+	//Get all the SNMPv3 User
+	allsnmpv3user, err1 := ovc.GetSNMPv3Users("", "", "", "")
+	if err1 != nil {
+		panic(err1)
+	} else {
+		fmt.Println("#-----Got All SNMPv1 Trap Destinations-----#")
+		for i := range allsnmpv3user.Members {
+			fmt.Println(allsnmpv3user.Members[i])
+		}
+	}
+
+	//Get SNMPv3 User by ID
+	id := "49a96a2a-a200-44df-b600-3a4c3d9f267c"
+	snmpv3User, err2 := ovc.GetSNMPv3UserById(id)
+	if err2 != nil {
+		panic(err2)
+	} else {
+		fmt.Printf("\n#-----Got SNMPv3 User of ID %v", id)
+		fmt.Println(snmpv3User)
+	}
+
+	//Get SNMPv3 User by username
+	username := "user47"
```

Comment:
Change to username := snmpv3User.UserName, so that no need to hardcode in many places.

Reply:
fixed